entities/common.go

```diff
@@ -1,8 +1,24 @@
 package entities
 
+import (
+	"encoding/json"
+	"errors"
+)
+
 // Cursor represents the pagination cursor on many responses.
 type Cursor struct {
 	Limit  uint64 `json:"limit"`
-	After  uint64 `json:"after"`
-	Before uint64 `json:"before"`
+	After  string `json:"after"`
+	Before string `json:"before"`
+}
+
+// Epoch represents time.
+type Epoch int64
+
+// MarshalJSON marshals Epoch instance into JSON.
+func (e *Epoch) MarshalJSON() ([]byte, error) {
```

Comment:
If you were going to keep this type, this method wouldn't be necessary because
encoding/json already handles casting like this.

In fact, the default behavior would be more performant because it likely re-uses
the underlying buffer.

Reply:
Ack. Tried util.UnixTimestamp and it worked fine.